gradle.properties

```diff
@@ -19,3 +19,6 @@ android.useAndroidX=true
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
don't include this here, make sure it goes in the gradle `android` block

Reply:
removed